ui/src/pages/backoffice/Loan/LoanDetails/components/ItemMetadata/ItemMetadata.js

```diff
@@ -51,18 +51,10 @@ export default class ItemMetadata extends Component {
         <Grid.Column>
           <Table basic="very" definition className="metadata-table">
             <Table.Body>
-              <Table.Row>
-                <Table.Cell>Document ID</Table.Cell>
```

Comment:
we do not show then anymore?

Reply:
True, I could have left the doc pid, but it anyway requires quite some refactoring the entire view... So we will work on that!